### DISCUSSION TITLE
Project 3 Question 2 Index Error

### DISCUSSION THREAD

**Discussion by ecieszynski** [cooperrc/computational-mechanics#56] [Q&A]
Keep getting this index error and not really sure why 

![image](https://user-images.githubusercontent.com/106542680/175100003-18364ebc-cc0f-4784-9ef2-af49a294c5d1.png)

**Comment by cooperrc** [comment]
the `np.where(...)` function is returning an array which you index with `[0]`, but then the second `[0]` is trying to access an element from an element of an array. 

```python
np.where(np.arange(10) < 1)
out: array([0])
np.where(np.arange(10) < 1)[0]
out: 0
np.where(np.arange(10) < 1)[0][0]
out: Error
```
